src/cls/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ install(TARGETS cls_sdk DESTINATION ${cls_dir})
 # cls_hello
 set(cls_hello_srcs hello/cls_hello.cc)
 add_library(cls_hello SHARED ${cls_hello_srcs})
+target_link_libraries(cls_hello $<$<PLATFORM_ID:Windows>:ceph-common>)
```

Comment:
IIRC, you are only porting the client side of Ceph, why bother fixing the server side?

Reply:
Some of the client code dependencies were cascading to that, I'll check if it's still the case.